exampleCourse/infoCourse.json

```diff
@@ -21,6 +21,7 @@
         {"name": "mwest", "color": "gray1", "description": "Question originally written by mwest"},
         {"name": "tbretl", "color": "blue1", "description": "Question originally written by tbretl"},
         {"name": "balamut2", "color": "orange1", "description": "Question originally written by balamut2"},
+        {"name": "mfsilva", "color": "gray1", "description": "Question originally written by mfsilva"},
```

Comment:
In the `string_element` branch you used `orange1` for your color, which is now causing a conflict. Can you change this one to `orange1`?